src/ui/Chart/wrappers/CypherChartWrapper.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import ChartSwitch from "./ChartSwitch";
+import { Cypher } from "../../../utils/Cypher";
+import * as simpleMappers from "../utils/simpleMappers";
+
+class CypherChartWrapper extends Component {
+  constructor(props) {
+    super(props);
+    this.data = [];
+  }
+  renderChart(res, tick) {
+    this.data =
+      this.props.chartType === "pie" || this.props.chartType === "doughnut"
+        ? simpleMappers.cypherResultToCircularChartData(
+            res,
+            this.data,
+            this.props
+          )
+        : simpleMappers.cypherResultToXYChartData(
+            res,
+            this.data,
+            tick,
+            this.props
+          );
+
+    return <ChartSwitch {...this.props} data={this.data} />;
+  }
+  render() {
+    return (
+      <Cypher
+        interval={this.props.refreshInterval || 2}
+        query={this.props.query}
```

Comment:
I think `params={this.props.params}` should be added tp the `<Cypher>` incovation.

Reply:
I guess we should probably pass on `driver` also to support those that don't use the `<DriverProvider/>`